test/test_issue833.py

```diff
@@ -0,0 +1,61 @@
+# test for https://github.com/RDFLib/rdflib/issues/833
+
+from rdflib import Graph, URIRef
+import unittest
+
+
+class TestIssue833(unittest.TestCase):
+
+    def setUp(self):
+        self.graph = Graph()
+
+        self.tarek = URIRef(u'tarek')
+        self.bob = URIRef(u'bob')
+        self.likes = URIRef(u'likes')
+        self.hates = URIRef(u'hates')
+        self.pizza = URIRef(u'pizza')
+        self.cheese = URIRef(u'cheese')
+
+        self.quads = []
+
+    def tearDown(self):
+        self.graph.close()
+
+    def prepareQuadsList(self):
+        tarek = self.tarek
+        bob = self.bob
+        likes = self.likes
+        hates = self.hates
+        pizza = self.pizza
+        cheese = self.cheese
+
+        self.quads.append((tarek, likes, pizza, self.graph))
+        self.quads.append((tarek, likes, cheese, self.graph))
+        self.quads.append((tarek, likes, bob, self.graph))
+        self.quads.append((bob, likes, cheese, self.graph))
+        self.quads.append((bob, hates, pizza, self.graph))
+
+    def testAddN(self):
+        asserte = self.assertEqual
+        tarek = self.tarek
+        bob = self.bob
+        likes = self.likes
+        hates = self.hates
+        pizza = self.pizza
+        cheese = self.cheese
```

Comment:
Recommend to just define these on the module level -- less code and no reason not to.

(Don't ask me why it's done that way in other tests. IDK. It's kinda dumb.)

Reply:
Thanks for all the suggested changes. I have added them, please have a look.

Reply:
Looks OK to me.

(I was actually recommending to define the `URIRef`s _outside_ of the class, like on line 7.)